crates/provider-sqldb-postgres/README.md

```diff
@@ -0,0 +1,180 @@
+# 🐘 SQL Database Postgres Provider
+
+This capability provider implements the [wasmcloud:sqldb-postgres](https://github.com/wasmCloud/interfaces/tree/main/keyvalue) WIT package, which enables SQL-driven database interaction with a [Postgres][postgres] database cluster.
```

Comment:
Link seems to point at a different interface?

Reply:
Ah thanks for noting this, I need to update that link -- we'll likely be publishing the WIT definition to another repo similar to how wasi-cloud does it, but we still need to discuss it a bit!

I'll leave myself a TODO so I don't forget your comment here

- [ ] fix the WIT interface link after WIT contract is iterated & published.